src/main/scala/ai/privado/utility/SQLParser.scala

```diff
@@ -205,10 +206,16 @@ object SQLNodeBuilder {
     builder.addEdge(tableNode, fileNode, EdgeTypes.SOURCE_FILE)
 
     queryModel.column.zipWithIndex.foreach { case (queryColumn: SQLColumn, columnIndex) =>
+      /* As queries from .sql files are processed individually,
+        an offset equal to the lineNumber of the query in the original file - 1 is added to the isolated column lineNumber */
+      val lineNumber = fileName match
+        case Some(f) if f.endsWith(".sql") => queryColumn.lineNumber + queryLineNumber - 1
+        case _                             => queryColumn.lineNumber
```

Comment:
I believe this will be needed across and not specifically for .sql files? Add a test case for this case as well

Reply:
Nope, just for .sql files. Other files ( ex -> schema.rb ) files give correct line numbers. They have their individual tests as well.

Reply:
schema.rb is parsed by ruby parser and not by this utility function

Reply:
Agreed, but all the columns and tables are finally passed to this `SQLNodeBuilder`.